### PR TITLE
Added extra type hints and changed a parameter

### PR DESCRIPTION
**Pull request by ranbir7**

### CODE REVIEW

**Comment by ranbir7** [comment]
I changed the variable: ndf to n_params cuz I couldnt find any variable ndf in the function..
@ConnorStoneAstro ?

**Comment by ranbir7** [comment]
did u check this? @ConnorStone628 ?

File: autoprof/fit/base.py [hunk 1]
```diff
@@ -124,4 +125,4 @@ def _f(x, nu):
 
             if res.success:
                 return res.x[0]
-        raise RuntimeError(f"Unable to compute Chi^2 contour for ndf: {ndf}")
+        raise RuntimeError(f"Unable to compute Chi^2 contour for n_params: {n_params}")
```
Comment (ConnorStoneAstro) [inline autoprof/fit/base.py]: `ndf` is standard terminology for "number of degrees of freedom", while `n_params` is also correct, `ndf` is the standard and I think we should keep it.

Comment (ranbir7) [reply autoprof/fit/base.py]: Okay...

File: autoprof/fit/base.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
-from time import time
-from typing import Any,Union,Sequence
-import torch
 import numpy as np
-from scipy.special import gammainc
+import torch
 from scipy.optimize import minimize
+from scipy.special import gammainc
+from time import time
+from typing import Any, Union, Sequence
```
Comment (ConnorStoneAstro) [inline autoprof/fit/base.py]: `time` and `typing` are base python modules and so should be at the top

Comment (ranbir7) [reply autoprof/fit/base.py]: Okayyy I'll be changing it...

**Review by ConnorStoneAstro — APPROVED** [review]
The updates look great!

**Pull request merged**